Update addon_aggregates views to fix table rename bug1579478

jasonthomas opened the pull request:
Update views to use the updated table name in https://github.com/mozilla/telemetry-airflow/pull/596.

jasonthomas commented:
@relud are you aware of any recent changes to the google credential configuration for CircleCI `build`?

jklukas commented:
> @relud are you aware of any recent changes to the google credential configuration for CircleCI `build`?

It changed ~2 weeks ago when we merged https://github.com/mozilla/bigquery-etl/pull/272 . We should think a bit more carefully about how we want that to work, since it's indeed a problem right now that PRs submitted from anyone not in the `telemetry` GitHub org fail CI.

relud commented:
> since it's indeed a problem right now that PRs submitted from anyone not in the `mozilla/telemetry` GitHub team fail CI.

I'm generally in favor of avoiding this by submitting PRs from non-fork branches. I don't see any problem with granting everyone in `kparlante-all` and `moconnor-all` write permissions to this repo (esp with master protected), I just don't know if there are github teams for that.

jklukas commented:
It's my understanding that moving to CircleCI contexts basically erases the problem of fork vs. non-fork, but shifts the problem to whether the user is in the associated GitHub team or not. It does seem like we should have a GitHub team that should roughly cover the people we'd want to be able to submit PRs here.

jasonthomas commented:
> @jasonthomas could you grant access to the data-eng-circleci-deployment context to the svcops team and if possible grant data-eng-circleci-tests access to the entire mozilla org?

FWIW I was already in the telemetry mozilla github group. 

I've created a new group named `dataops` and added it to `data-eng-circleci-deployment` context and added 'All members' of mozilla github org to`data-eng-circleci-tests`. It looks like adding groups to the security context is not sufficient, I believe we will also need `Pass secrets to builds from forked pull requests` enabled on the CircleCI repo configuration. 

I think the next steps would be - 

* Move all current environment variables into restricted contexts, specifically DOCKERHUB related credentials. 
* Specify security context for each job in CircleCI workflow configuration
* Enable `Pass secrets to builds from forked pull requests`

Based on these changes I believe the forked pull requests will receive the correct environment variables based restricted context membership.

jklukas commented:
I am wary of turning on `Pass secrets to builds from forked pull requests` since it relies on discipline to make sure we put secrets only in CircleCI contexts and don't add any secrets directly to the job.

Other than that, @jasonthomas's proposed steps sound like they should work. I particularly like the idea of a future where only the job that does Docker releases has DockerHub creds, etc.
